webextension/background.js

```diff
@@ -129,8 +138,90 @@ const Config = (function() {
       }
     }
 
-    load() {
-      return Promise.all([
+    maybeSendTelemetry(message) {
+      if (!this._telemetryEnabled) {
+        return false;
+      }
+
+      browser.study.sendTelemetry(message);
+      return true;
+    }
+
+    get shieldStudySetup() {
+      const pref = `extensions.${browser.runtime.id.split("@")[0]}.variation`;
+      return {
+        activeExperimentName: browser.runtime.id,
+        allowEnroll: true,
+        studyType: "shield",
+        telemetry: {
+          send: true,
+          removeTestingFlag: !this._testingMode,
+        },
+        endings: {
+          "user-disable": {
+            baseUrl: null,
+          },
+          expired: {
+            baseUrl: null,
+          },
+          dataPermissionsRevoked: {
+            baseUrl: null,
+            study_state: "ended-neutral",
+          },
+        },
+        logLevel: 10,
+        variationOverridePreference: pref,
+        weightedVariations: UIVariants.map(name => {
+          return {name, weight: 1};
+        }),
+        expire: {
+          days: 60,
```

Comment:
That was just me putting in a big number to catch your attention, since I really wasn't sure if we wanted it to be for 15 days or "indefinitely" until we shut off the experiment at some point (since we want the icon to continue to be usable by users even if they're not auto-prompted).

Reply:
Heh. Let's set it to 14 please.